### ISSUE TITLE
Homepage alert margins need adjusting

### ISSUE THREAD

**Issue by mxsnyder** [baltimorecounty/baltimorecountymd.gov-assets#144]

**Comment by mxsnyder** [comment]
From Sam:
```CSS
@only screen and (max-width: 400px)
.search .search-input {

    height: auto;
    padding: 10px 10px;

}

@only screen and (max-width: 992px)
.carousel-wrapper .alert {
    margin: -10px 20px 0;
}
```

**Comment by mxsnyder** [comment]
Fixed in release v1.9.3.
